docs/clusterdefinition.md

```diff
@@ -173,7 +173,8 @@ Below is a list of kubelet options that acs-engine will configure by default:
 |"--cluster-domain"|"cluster.local"|
 |"--pod-infra-container-image"|"pause-amd64:*version*"|
 |"--max-pods"|"30", or "100" if using kubenet --network-plugin (i.e., `"networkPlugin": "kubenet"`)|
-|"--eviction-hard"|"memory.available<100Mi,nodefs.available<10%,nodefs.inodesFree<5%"|
+|"--eviction-hard"|"memory.available<5%,nodefs.available<10%,nodefs.inodesFree<5%"|
+|"--kube-reserved"|"cpu=1,ephemeral-storage=1Gi"|

```

Comment:
There is a reference in GKE docs: https://cloud.google.com/kubernetes-engine/docs/concepts/cluster-architecture#node_allocatable